vignettes/datatable-programming.Rmd

```diff
@@ -456,6 +456,50 @@ DT[, eval(cl)]
 DT[, cl, env = list(cl = cl)]
 ```
 
+## Error handling with classed conditions
```

Comment:
I'm not sure this is the most natural fit for this vignette, nor do I have a good answer for which vignette it fits in / if it warrants a new vignette.

I wonder if this should instead be a new Rd page, like the datatable-options Rd page, with an associated code-quality check to make sure this is in sync with the current offerings. WDYT @tdhock @jangorecki?

Reply:
programming seems ok to me

Reply:
programming vignette is at the moment in 100% about customizing queries with character strings. I think it is best to keep it as a separate manual entry.